README.md

```diff
@@ -122,7 +123,7 @@ on a specific path, and are stored on the host filesystem without any
 sort of diff mechanism (that is, any file hierarchy data written to the volume
 is always available in that hierarchy).
 
-For single service applications (ie. non-multicontainer applications without
+For single service applications (eg. non-multicontainer applications without
```

Comment:
```suggestion
For single service applications (e.g. non-multicontainer applications without
```
We don't have a style guide so 🤷‍♂ but e.g. would be preferable if we had one :)

Reply:
Updated!